lib/Backends/CPU/tests/CPUOperatorTest.cpp

```diff
@@ -46,6 +46,11 @@ std::set<std::string> glow::backendTestBlacklist = {
     "batchedPairwiseDotProduct/0",
     "batchedReduceAdd_BFloat16/0",
     "batchedReduceAdd_Float16/0",
+    "batchedReduceProd_Float16/0",
+    "batchedReduceProd_BFloat16/0",
+    "batchedReduceProd_Int32/0",
+    "batchedReduceProd_Int64/0",
+    "batchedReduceProd_Int8/0",
```

Comment:
Still confused here -- there aren't any such `Float16`/`BFloat16`/`Int8` tests in OperatorTests. Can you make sure that we have sufficient coverage in OperatorTests to cover new functionality/kernels in all backends, and then only blacklist the actually existing tests?

Reply:
CPU supports only Float32, Added the changes accordingly.